sass/_moj-overrides.scss

```diff
@@ -1,3 +1,10 @@
 .moj-pagination__results {
   margin: 0;
 }
+
+.moj-action-bar__filter {
+  float: left;
+  padding-right: 10px;
+}
+
```

Comment:
required, else pagination and this button appear on different lines - too much wasted space